sky/skylet/job_lib.py

```diff
@@ -291,7 +291,7 @@ def get_latest_job_id() -> Optional[int]:
 
 
 def get_job_time_payload(job_id: int, is_end: bool) -> Optional[int]:
```

Comment:
Can we rename this? `get_job_submitted_or_ended_timestamp_payload`? "job time" sounds like a duration, but here we want one of the two timestamps.

Need to update https://github.com/skypilot-org/skypilot/blob/8a80f5be49d951d1e6334f2cd1e6595f65e4e413/sky/spot/spot_utils.py#L123 comment to say submitted_at?  

To keep this interface consistent with that func, is_end -> get_end_time?